zokrates_core/src/parser/tokenize/tokenizer.rs

```diff
@@ -201,6 +201,24 @@ pub fn next_token<T: Field>(input: &String, pos: &Position) -> (Token<T>, String
                 },
             ),
         },
+        Some('&') => match input.chars().nth(offset + 1) {
+            Some('&') => (
+                Token::AndAnd,
+                input[offset + 2..].to_string(),
+                Position {
+                    line: pos.line,
+                    col: pos.col + offset + 2,
+                },
+            ),
+            _ => (
```

Comment:
Why do we have logic to tokenize bitwise And in this PR, but no further compiling? I'm assuming it's to make this pattern matching exhaustive, but I think we might be better off throwing an error than writing a half-implemented feature. It would make more sense to have all the logic for bitwise And implemented together in a future PR

Reply:
Good catch! I agree we should get rid of `&` for now

Reply:
This was fixed and there's no and operator now